src/choropleth.ts

```diff
@@ -834,9 +836,19 @@ function showMapCallout(data, event, d): void {
 
   callout.style("display", "block");
   const boundingRect: DOMRect = callout.node().getBoundingClientRect();
-  callout
-    .style("left", event.pageX - boundingRect.width / 2 + "px")
-    .style("top", event.pageY - boundingRect.height - 20 + "px");
+
+  // Base callout location on the screen dimenstions
+  if (window.innerWidth > mobileScreenWidth) {
```

Comment:
Does "auto" not work well for desktop?

Reply:
Not really, it works, but doesn't look as good, shows up further away than wanted